demo/components/popup-demo.tsx

```diff
@@ -1,38 +1,88 @@
 import React = require('react');
-import {Popup} from '../../src/';
+import { HorizontalPosition, Popup, PositionPoint, RadioButton, RadioGroup, VerticalPosition } from '../../src/';
```

Comment:
Please refactor names like `HorizontalPosition` to `PopupHorizontalPosition` to be more clear where things come from.